BE/OwnershipAndControl/CorporateControl.rdf

```diff
@@ -122,7 +92,32 @@
 				</owl:unionOf>
 			</owl:Class>
 		</owl:equivalentClass>
-		<skos:definition>legal person that directly, or indirectly through one or more intermediaries, controls, or is controlled by, or is under common control with, another legal person</skos:definition>
+		<skos:definition>party that is related to a legal entity, directly, or indirectly through one or more intermediaries, and controls, or is controlled by, or is under common control with that entity, typically determined by the degree of ownership</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-be-oac-cctl;Affiliation">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-oac-ctl;Control"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasActor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-oac-cctl;ControllingAffiliate"/>
```

Comment:
If 2 entities are affiliates through having a common parent, then neither one controls the other; it's a symmetric relationship.